core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -1280,8 +1293,8 @@ private[deploy] class Master(
     for (worker <- toRemove) {
       if (worker.state != WorkerState.DEAD) {
         val workerTimeoutSecs = TimeUnit.MILLISECONDS.toSeconds(workerTimeoutMs)
-        logWarning("Removing %s because we got no heartbeat in %d seconds".format(
-          worker.id, workerTimeoutSecs))
+        logWarning(log"Removing ${MDC(LogKeys.WORKER_ID, worker.id)} because we got no heartbeat " +
+          log"in ${MDC(LogKeys.TIME_UNITS, workerTimeoutSecs)} seconds")
```

Comment:
All the time units are now using ms instead of seconds. Shall we just use `ms` here?